taskfiles/dev.yml

```diff
@@ -36,6 +36,14 @@ tasks:
     status:
     - test -f '{{.VENV}}/bin/yapf'
 
+  install-go:
+    desc: install golang compiler
+    cmds:
+    - mkdir -p '{{.BUILD_ROOT}}/go/'
+    - curl -sSLf --retry 3 --retry-connrefused --retry-delay 2 'https://storage.googleapis.com/vectorizedio-public/dependencies/go1.16.5.linux-{{ARCH}}.tar.gz' | tar -xz -C '{{.BUILD_ROOT}}/go' --strip 1
```

Comment:
for OSS we should use upstream go; 

for our build we can use our bucket.

Reply:
sounds good